examples/carousel/index.html

```diff
@@ -74,44 +74,172 @@
+              <a href="" class="btn text-center text-uppercase Stories-more">Read the Story</a>
+            </article>
+          </div>
+          <div class="col-2 col-sm-12 col-md-6 col-lg-2">
+            <article class="Stories-item">
+              <blockquote class="Stories-quote">
+                <p class="heading-4 spacing-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam consectetur venenatis ante, vel venenatis magna placerat et. Praesent molestie velit vitae ante ultrices tempus.</p>
+                <img src="https://api.adorable.io/avatars/285/eric%40firstandthird.com" class="spacing-sm Stories-avatar">
+                <footer class="Stories-byline">
+                  Eric Willis<br />
+                  Cal Poly SLO
+                </footer>
+              </blockquote>
+              <a href="" class="btn text-center text-uppercase Stories-more">Read the Story</a>
+            </article>
+          </div>
+          <div class="col-2 col-sm-12 col-md-6 col-lg-2">
+            <article class="Stories-item">
+              <blockquote class="Stories-quote">
+                <p class="heading-4 spacing-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam consectetur venenatis ante, vel venenatis magna placerat et. Praesent molestie velit vitae ante ultrices tempus.</p>
+                <img src="https://api.adorable.io/avatars/285/michael%40firstandthird.com" class="spacing-sm Stories-avatar">
+                <footer class="Stories-byline">
+                  Michael Turnwall<br />
+                  UC San Diego
+                </footer>
+              </blockquote>
+              <a href="" class="btn text-center text-uppercase Stories-more">Read the Story</a>
+            </article>
+          </div>
+          <div class="col-2 col-sm-12 col-md-6 col-lg-2">
+            <article class="Stories-item">
+              <blockquote class="Stories-quote">
+                <p class="heading-4 spacing-sm">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam consectetur venenatis ante, vel venenatis magna placerat et. Praesent molestie velit vitae ante ultrices tempus.</p>
+                <img src="https://api.adorable.io/avatars/285/newton%40firstandthird.com" class="spacing-sm Stories-avatar">
+                <footer class="Stories-byline">
+                  Newton Koumantzelis<br />
+                  USC
+                </footer>
+              </blockquote>
+              <a href="" class="btn text-center text-uppercase Stories-more">Read the Story</a>
+            </article>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="Stories">
+      <div class="container">
+        <div class="row">
+          <header class="col-12">
+            <h1 class="Stories-title text-center heading-2 spacing-lg">Carousel Peekaboo</h1>
+          </header>
+        </div>
+        <div class="row row-sm-carousel row-md-carousel carousel-peekaboo">
```

Comment:
I'd vote for changing `carousel-peekaboo` to `row-carousel-peekaboo` to keep it namespaced in the same family.